### PR TITLE
Update profile

### PR DESCRIPTION
**Pull request by tresoldi**
As discussed at https://github.com/lexibank/cals/issues/1

I fixed all BIPA issues and corrected two errors that looked like artefacts of copy&paste. Please review (especially @LinguList )

### CODE REVIEW

**Comment by tresoldi** [comment]
Just noticed that, once more, my setup is removing the language information (such as lat&long). Will check again, not sure was it is happening.

**Comment by chrzyki** [comment]
I think your pyglottolog is outdated (with 2.2.x), should be 3.0.0.

**Comment by tresoldi** [comment]
Yes, it is cascading because the default Python version in my system is 3.5, and it downgraded everything. I am building a new release environment now. Thanks @chrzyki

File: cldf/forms.csv [hunk 1]
```diff
@@ -138,7 +138,7 @@ kkhalqabad1-tolaugh-1,,kkhalqabad1,tolaugh,kʉlʊː,kʉlʊː,k ʉ l ʊː,,Mennec
 kkhalqabad1-tolie-1,,kkhalqabad1,tolie,ʒatʊː,ʒatʊː,ʒ a t ʊː,,Mennecier2016,,
 kkhalqabad1-tolive-1,,kkhalqabad1,tolive,ʒasaʊ,ʒasaʊ,ʒ a s a ʊ,,Mennecier2016,,
 kkhalqabad1-toplay-1,,kkhalqabad1,toplay,wɔjnaʊ,wɔjnaʊ,w ɔ j n a ʊ,,Mennecier2016,,
-kkhalqabad1-topull-1,,kkhalqabad1,topull,itʲerʊ,itʲerʊ,i tʲ e r ʊ,,Mennecier2016,,
+kkhalqabad1-topull-1,,kkhalqabad1,topull,sɔzʊ,sɔzʊ,s ɔ z ʊ,,Mennecier2016,,
```
Comment (LinguList) [inline cldf/forms.csv]: Important: this is a change of the entire form for the same language, right? Can you please check what is the reason for this change, @tresoldi ?

**Review by LinguList — COMMENTED** [review]
## Requests to check

- [x]  the line `kkhalqabad1-topull-1,,kkhalqabad1,topull,sɔzʊ,sɔzʊ,s ɔ z ʊ,,Mennecier2016,, ` is completely different from the earlier line we had ` kkhalqabad1-topull-1,,kkhalqabad1,topull,itʲerʊ,itʲerʊ,i tʲ e r ʊ,,Mennecier2016,, `, so the forms are now replaced by some new form, can you please search for the reason?

- [x] once we are already working on the IDs, could you make sure the language IDs that they start with a capital letter? I'd prefer that for readability, etc.

**Comment by tresoldi** [comment]
@chrzyki should be correct now.

File: cldf/forms.csv (hunk 1 above)
Comment (tresoldi) [reply cldf/forms.csv]: Wow, @LinguList , that was a good catch!

So, there are two columns glossed as `to pull (тянуть)` in the [raw data](https://github.com/lexibank/cals/blob/master/raw/Table_S2_Supplementary_Mennecier_et_al..docx): the first starting on p.46, the second on p. 54 . Both `itʲerʊ` and `sɔzʊ` would be right, the first in the `raw/12.csv` file, the second in the `raw/14.csv` one. The difference is due to reproducibility: as `self.raw_dir.glob(".csv")` returns an unsorted list, different words will be the selected synonym each time we run the script. The fix is obvious, just sort the list.

However, it seemed strange to have these repeated glosses, so I went to check with Kazakh and Uzbek dictionaries. Sure enough, the `itʲerʊ` set has a wrong gloss in the supplementary material, because it actually means "to push" (a concept otherwise missing). Even Google Translate has it, see [итеру](https://translate.google.com/?hl=en#view=home&op=translate&sl=kk&tl=en&text=%D0%B8%D1%82%D0%B5%D1%80%D1%83).

What should we do?

Comment (LinguList) [reply cldf/forms.csv]: I'd say: first compare with the doc-file, what they have there, as the raw.csv is derived from this, and then make a decision that can be pragmatic. If you think a gloss is wrong, I trust you on this, and you could ignore it, or replace it, etc.

Comment (tresoldi) [reply cldf/forms.csv]: Yes, it is wrong in the doc-file, I also checked on Brill's website.

I will correct the gloss and sort the list returned by `glob` nonetheless.

**Comment by tresoldi** [comment]
Last commit fixes the issues with the wrong gloss and capitalizes the languages IDs.

**Review by LinguList — APPROVED** [review]

**Review by chrzyki — APPROVED** [review]
Looks good. Just needs fixes for the test assertions.

**Comment by tresoldi** [comment]
@chrzyki should be fixed now.
